src/Utils.php

```diff
@@ -122,20 +122,20 @@ static function (int $a, int $b): int {
     }
 
     /**
-     * Join names in natural language wrapped in backticks, e.g. `a`, `b` and `c`.
+     * Join names in natural language wrapped in string $wrapper.
      *
      * @param string[] $names
```

Comment:
```suggestion
     * @param string[]         $names
     * @param non-empty-string $wrapper
```

We probably need assertion for non-empty string here, OR test case that ensure it can also properly print natural sentence like `foo, bar and baz` (without any wrapper). Maybe we should even check if `$wrapper` is a single-char string, because probably there should not be a use case for joining names like `WRAPfooWRAP, WRAPbarWRAP and WRAPbazWRAP` 😉.

Reply:
I opted to allow empty strings because I remember I saw an implode call within the project without any wrapper. Anyway, I added tests using empty string and test checking if an error occurs if multi-char string given.